packages/generator-electrode/component-add/index.js

```diff
@@ -104,13 +104,26 @@ module.exports = generators.Base.extend({
     },
 
     askFor: function() {
+      function randomComponentName() {
```

Comment:
it'd be a bit more consistent to use `untitled###` where `###` is a random number.